beta/workspace-settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "actor":"admin.email.goes.here@example.com",
+    "domain":"example.1password.om",
```

Comment:
Small typo here. I assume `example.1password.com` was intended.

Perhaps some commments would also be helpful here throughout, e.g.:
```
    "domain":"example.1password.com",            // the sign-in address of your 1Password account
```